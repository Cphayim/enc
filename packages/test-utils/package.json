{
  "name": "@cphayim-enc/test-utils",
  "private": true,
  "description": "",
  "keywords": [],
  "license": "ISC",
  "type": "module",
  "main": "src/index.ts"
}
