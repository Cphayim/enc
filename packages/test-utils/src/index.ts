import { vi } from 'vitest'

export function createConsoleSpy() {
  return {
    log: vi.spyOn(console, 'log').mockImplementation(() => void 0),
    debug: vi.spyOn(console, 'debug').mockImplementation(() => void 0),
    info: vi.spyOn(console, 'info').mockImplementation(() => void 0),
    warn: vi.spyOn(console, 'warn').mockImplementation(() => void 0),
    error: vi.spyOn(console, 'error').mockImplementation(() => void 0),
  }
}
