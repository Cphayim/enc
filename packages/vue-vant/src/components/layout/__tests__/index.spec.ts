import { expect, it } from 'vitest'
import { mount } from '@vue/test-utils'

import { EncLayout } from '..'

it('should be able to mount component', () => {
  const wrapper = mount(EncLayout, { props: { msg: '124' } })
  expect(wrapper.html()).toMatchSnapshot()
  expect(wrapper.get('div').text()).toBe('124')
})
