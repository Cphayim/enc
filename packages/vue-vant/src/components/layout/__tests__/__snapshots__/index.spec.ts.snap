// Vitest Snapshot v1

exports[`should be able to mount component 1`] = `"<div>124</div>"`;
